### PR TITLE
Update mfa.md

### LINKED ISSUE #30240: Entra rename
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ebdbdab0-c474-3fdd-76aa-ade905e76079
* Version Independent ID: 761ea2cc-7e05-4ae0-0875-44061a163621
* Content: [Multi-factor authentication in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/mfa?view=aspnetcore-7.0)
* Content Source: [aspnetcore/security/authentication/mfa.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authentication/mfa.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @damienbod
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #30240

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/authentication/mfa.md](https://github.com/dotnet/AspNetCore.Docs/blob/611ef015e05be1582684ce5f4aff41b40521f39a/aspnetcore/security/authentication/mfa.md) | [Multi-factor authentication in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/security/authentication/mfa?branch=pr-en-us-30241) |

### CODE REVIEW

File: aspnetcore/security/authentication/mfa.md [hunk 1]
```diff
@@ -49,11 +49,11 @@ To disable support for MFA TOTP, configure authentication using <xref:Microsoft.
 passkeys/FIDO2 is currently:
 
 * The most secure way of achieving MFA.
-* MFA that protects against phishing attacks. (As well as certificiate authentication and Windows for business)
+* MFA that protects against phishing attacks. (As well as certificate authentication and Windows for business)
 
 At present, ASP.NET Core doesn't support passkeys/FIDO2 directly. Passkeys/FIDO2 can be used for MFA or passwordless flows.
 
-Azure Active Directory provides support for passkeys/FIDO2 and passwordless flows. For more information, see [Passwordless authentication options for Azure Active Directory](/azure/active-directory/authentication/concept-authentication-passwordless).
+Microsoft Entra IDprovides support for passkeys/FIDO2 and passwordless flows. For more information, see [Passwordless authentication options](/azure/active-directory/authentication/concept-authentication-passwordless).
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/mfa.md:56]:
```suggestion
Microsoft Entra ID provides support for passkeys/FIDO2 and passwordless flows. For more information, see [Passwordless authentication options](/azure/active-directory/authentication/concept-authentication-passwordless).
```

File: aspnetcore/security/authentication/mfa.md [hunk 2]
```diff
@@ -550,11 +550,11 @@ See the following link for implementation details:
 passkeys/FIDO2 is currently:
 
 * The most secure way of achieving MFA.
-* MFA that protects against phishing attacks. (As well as certificiate authentication and Windows for business)
+* MFA that protects against phishing attacks. (As well as certificate authentication and Windows for business)
 
 At present, ASP.NET Core doesn't support passkeys/FIDO2 directly. Passkeys/FIDO2 can be used for MFA or passwordless flows.
 
-Azure Active Directory provides support for passkeys/FIDO2 and passwordless flows. For more information, see [Passwordless authentication options for Azure Active Directory](/azure/active-directory/authentication/concept-authentication-passwordless).
+Microsoft Entra IDprovides support for passkeys/FIDO2 and passwordless flows. For more information, see [Passwordless authentication options for Azure Active Directory](/azure/active-directory/authentication/concept-authentication-passwordless).
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/mfa.md:557]:
```suggestion
Microsoft Entra ID provides support for passkeys/FIDO2 and passwordless flows. For more information, see [Passwordless authentication options](/azure/active-directory/authentication/concept-authentication-passwordless).
```

File: aspnetcore/security/authentication/mfa.md [hunk 3]
```diff
@@ -49,11 +49,11 @@ To disable support for MFA TOTP, configure authentication using <xref:Microsoft.
 passkeys/FIDO2 is currently:
 
 * The most secure way of achieving MFA.
-* MFA that protects against phishing attacks. (As well as certificiate authentication and Windows for business)
+* MFA that protects against phishing attacks. (As well as certificate authentication and Windows for business)
```
Comment (tdykstra) [inline aspnetcore/security/authentication/mfa.md:52]: This is a preexisting problem, but the "(As well as ...)" sentence doesn't make sense in context. It should probably be made into two extra bullets but then it still wouldn't make sense without something additional.

**Review by tdykstra — APPROVED** [review]
LGTM, just a comment to think about

**Pull request merged**